parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -487,6 +487,25 @@ configAddons() {
 }
 {{end}}
 {{- if HasNSeriesSKU}}
+
+installGPUDriversRun() {
```

Comment:
Why do we need to do this sometimes?

Reply:
the RCA is not yet very clear, it should be the issue of the driver installer.
when the first try of installation failed, it will leave a dirty directory there, and the retry will not works if there's dirty directory.